### PR TITLE
Additional Playbook Migrations

### PR DESCRIPTION
**Pull request by ljstella**
- [x] trustar_enrich_indicators
- [x] threat_intel_investigate
- [x] start_investigation
- [x] aws_disable_user_accounts
- [x] aws_find_inactive_users

### CODE REVIEW

File: docs/_pages/playbooks.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ sidebar:
 
 | Name    | Detections | Type        |
 | --------| ---------- | ----------- |
+| [AWS Disable User Accounts](/playbooks/aws_disable_user_accounts/)| None | Input |
```
Comment (philroyer-phantom) [inline docs/_pages/playbooks.md]: There seem to be two kinds of "types". There is input vs automation, and investigation vs response. To be consistent, this one should be "Response" I think.

File: docs/_pages/playbooks.md [hunk 2]
```diff
@@ -36,3 +38,6 @@ sidebar:
 | [Risk Notable Protect Assets and Users](/playbooks/risk_notable_protect_assets_and_users/)| None | Response |
 | [Risk Notable Review Indicators](/playbooks/risk_notable_review_indicators/)| None | Response |
 | [Risk Notable Verdict](/playbooks/risk_notable_verdict/)| None | Response |
+| [Start Investigation](/playbooks/start_investigation/)| None | Investigation |
+| [Threat Intel Investigate](/playbooks/threat_intel_investigate/)| None | Investigation |
+| [TruSTAR Enrich Indicators](/playbooks/trustar_enrich_indicators/)| None | Input |
```
Comment (philroyer-phantom) [inline docs/_pages/playbooks.md]: Type should be Investigation I think.

File: docs/_playbooks/trustar_enrich_indicators.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+title: "TruSTAR Enrich Indicators"
+last_modified_at: 2021-11-24
+toc: true
+toc_label: ""
+tags:
+  - Input
+  - Splunk SOAR
+  - TruSTAR
```
Comment (philroyer-phantom) [inline docs/_playbooks/trustar_enrich_indicators.md:9]: Could we please add "Investigate" and "Threat Intel" here as tags to be consistent with the platform?

File: docs/index.markdown [hunk 4]
```diff
@@ -28,7 +28,7 @@ feature_row:
   - image_path: /static/feature_playbooks.png
     alt: "100% free"
     title: "Playbooks"
-    excerpt: "See all **26** sets of steps 🐾 to automatically response to a threat."
+    excerpt: "See all **31** sets of steps 🐾 to automatically response to a threat."
```
Comment (philroyer-phantom) [inline docs/index.markdown]: Can we please replace this with:

See all **31** automated investigation and response playbooks.

Or something like that?

**Review by philroyer-phantom — DISMISSED** [review]
Overall looks good, but had some minor fixups to make if possible.

File: docs/_pages/playbooks.md (hunk 1 above)
Comment (ljstella) [reply docs/_pages/playbooks.md]: FIxed the Input playbooks to fit in Investigation vs Response.

File: docs/_pages/playbooks.md (hunk 2 above)
Comment (ljstella) [reply docs/_pages/playbooks.md]: FIxed the Input playbooks to fit in Investigation vs Response.

File: docs/_playbooks/trustar_enrich_indicators.md (hunk 3 above)
Comment (ljstella) [reply docs/_playbooks/trustar_enrich_indicators.md:9]: I added all of the tags.platform_tags from the playbook YAMLs to populate the tags section of each page.

File: docs/index.markdown (hunk 4 above)
Comment (ljstella) [reply docs/index.markdown]: Modified language here.

**Review by philroyer-phantom — DISMISSED** [review]
Looks good!

**Review by patel-bhavin — APPROVED** [review]

**Pull request merged**